middlewares/validators/events.js

```diff
@@ -100,11 +100,57 @@ const endActiveEvent = async (req, res, next) => {
   }
 };
 
+const addPeerToEvent = async (req, res, next) => {
+  const { id } = req.params;
+  const { peerId, name, role, joinedAt } = req.body;
+
+  const schema = joi.object({
+    peerId: joi.string().required(),
+    name: joi.string().required(),
+    id: joi.string().required(),
+    role: joi.string().required(),
+    joinedAt: joi.date().required(),
+  });
+
+  const validationOptions = { abortEarly: false };
+
+  try {
+    await schema.validateAsync({ peerId, name, id, role, joinedAt }, validationOptions);
```

Comment:
is there a chance of schema validation to fail?

Reply:
Yes, it will. If any of the required fields are missing or not provided or if the provided data types for the fields do not match the expected types the validation will fail.